db/schema.sql

```diff
@@ -8216,39 +8216,6 @@ CREATE SEQUENCE maker.clip_dog_id_seq
 ALTER SEQUENCE maker.clip_dog_id_seq OWNED BY maker.clip_dog.id;
 
 
---
--- Name: clip_ilk; Type: TABLE; Schema: maker; Owner: -
---
-
-CREATE TABLE maker.clip_ilk (
```

Comment:
Looks like `clip_ilk` disappeared from the schema. Is that expected? I don't see any changes in here related to that table.

Reply:
That might be the result of my change from last time around. vat and ilk are immutable so not necessary to keep track of. I had them in the schema with earlier static fields.